jme3-core/src/plugins/java/com/jme3/texture/plugins/TGALoader.java

```diff
@@ -71,7 +77,18 @@
     // 11 - run-length encoded, black and white image
     public static final int TYPE_BLACKANDWHITE_RLE = 11;
 
+    protected static TGALoader loader;
```

Comment:
Loaders are typically managed via `AssetManager`, there's no need to have a static reference in any `AssetLoader`.